webapp/src/debugger.tsx

```diff
@@ -290,6 +290,7 @@ export class DebuggerToolbar extends data.Component<DebuggerToolbarProps, Debugg
 
         const simState = parentState.simState;
         const isRunning = simState == pxt.editor.SimState.Running;
+        const isStarting = simState == pxt.editor.SimState.Starting;
```

Comment:
There is also a pending state where the editor is transitioning between editors.

Reply:
Yes, but we are not allowing to change editors while in debug mode